charts/posthog/README.md

```diff
@@ -283,9 +379,29 @@ If running the command shows the SSL cert as `PROVISIONING`, that means that the
 
 As a troubleshooting tool, you can allow HTTP access by setting `ingress.gcp.forceHttps` and `web.secureCookies` both to false, but we recommend always accessing PostHog via https.
 
+</details>
+
+<details>
+  <summary>
+    <b>Using Nginx Ingress Controller</b>
+  </summary>
+
+<br />
+
+As a troubleshooting tool, you can allow HTTP access by setting these values in your `values.yaml`, but we recommend always accessing PostHog via https.
+```
+ingress:
+  redirectToTLS: false
+  letsencrypt: false
+web:
+  secureCookies: false
+```
+
+</details>
+
 ### How can I connect to my ClickHouse instance?
 
-- Get the IP via `kubectl get svc`
+- Get the IP via `kubectl get svc --namespace posthog`
```

Comment:
This change is wrong: Other commands here don't encourage specifying the namespace. Users can also specify any namespace they'd like - `--namespace posthog` varies from one to the other.